spec/factories/mission.rb

```diff
@@ -7,5 +7,6 @@
     started_at { DateTime.now }
     ended_at { DateTime.now + 1 }
     created_at { DateTime.now }
+    user_id { User.last.id }
```

Comment:
https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations

```suggestion
    association :user
```